gruntfile.js

```diff
@@ -201,8 +203,16 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-contrib-stylus');
     grunt.loadNpmTasks('grunt-autoprefixer');
     grunt.loadNpmTasks('grunt-gh-pages');
+    
+    grunt.registerTask('grok', 'Fire up ngrok so icanhaz https.', function () {
```

Comment:
although it's a nice play on words, i think naming the task `ngrok` would be clearer